iocore/eventsystem/SocketManager.cc

```diff
@@ -108,3 +111,36 @@ SocketManager::close(int s)
   } while (res == -EINTR);
   return res;
 }
+
+bool
+SocketManager::fastopen_supported()
```

Comment:
Ah. I would say then that it shouldn't be a method, it should be a file scoped function called by reconfigure. I don't think we want to be checking on every open.